courses/free-courses-de.md

```diff
@@ -27,6 +29,17 @@
 * [Haskell Tutorials Deutsch](https://www.youtube.com/playlist?list=PLNmsVeXQZj7pFIXDN1NLw6jMExuK-wN8I) - The Morpheus Tutorials
 
 
+### HTML and CSS
+
+* [CSS lernen](https://youtube.com/playlist?list=PLuBK_vNnGp8ANspdZh_aRAa1InIhFlgm_) - NEW - Vadim
+* [HTML Tutorial Deutsch](https://youtube.com/playlist?list=PLnlqg5o1zhnhVI3t1iTE2oO4QSGpu7EMx) - Markus Reichl
+
+
+#### Bootstrap
+
+* [Bootstrap 4 Tutorial (Deutsch)](https://youtube.com/playlist?list=PLiH_qbTmMNfhcZazOxjK9hFdtRfq_NLld) - Matthias Dangl
```

Comment:
It's usually not helpful to add a category or subcategory for a single resource. Usually, we try to find another category (such as `HTML and CSS`, or see if there are other books in our lists that would fit into your category. This is a judgement call - I'm going to allow it here for now, but what we don't want to see is a long list of single entry subcategories for every CSS framework on the planet.

Reply:
> 

Ok